tests/FamilyHubs.ReferralUi.UnitTests/Services/WhenUsingOpenReferralOrganisationClientService.cs

```diff
@@ -17,13 +18,13 @@ public async Task ThenGetTaxonomyList()
             new TaxonomyDto(
                         "UnitTest bccsource:Organisation",
                         "Organisation",
-                        "Test BCC Data Sources",
+                        TaxonomyType.ServiceCategory,
```

Comment:
this tests are not really testing anything and on top of that the method under test is not used anywhere in the project

Reply:
Removed redundant code.